tjenestebuss-integrasjon/.nais/prod.yaml

```diff
@@ -0,0 +1,38 @@
+app-name: "pensjonsbrev-tjenestebuss"
+ingresses:
+  - "https://pensjonsbrev-tjenestebuss.intern.nav.no"
```

Comment:
Hvorfor har vi 2 ingresser egentlig? Ser at gcp varianten for dev brukes for lokal, skal vi ha det for prod og?

Reply:
Godt poeng!